resourcemanager/recaser/registration.go

```diff
@@ -0,0 +1,31 @@
+package recaser
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonids"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/resourceids"
+)
+
+var KnownResourceIds = make(map[string]resourceids.ResourceId)
```

Comment:
Out of interest, what's the reason for having this public/exported?

Reply:
the compiler was giving an error, but you're right it looks like it doesn't really need to be public - fixed